composer.json

```diff
@@ -115,6 +115,7 @@
         "drupal/restui": "^1.21",
         "drupal/search_api": "^1.31",
         "drupal/twig_tweak": "^3.2.1",
+        "drupal/twigsuggest": "^1.0@RC",
```

Comment:
This is nice. 

It also makes {{base_path}} available in twig templates

Reply:
I also like it but we decided against (for now) in [20e8df9](https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/665/commits/20e8df9caccaf6388cbb60a08a4376ebd0c80727).